inc/widgets/views/account.php

```diff
@@ -0,0 +1,246 @@
+<?php
+/**
+ * Account Widget View
+ *
+ * This is rendered on the WordPress dashboard Bluehost Account widget.
+ * Icon svgs from HeroIcons https://heroicons.com/
+ *
+ * @package WPPluginBluehost
+ */
+
+use function NewfoldLabs\WP\ModuleLoader\container;
+use function NewfoldLabs\WP\Context\getContext;
+
+// Helper methods for links : move these elsewhere so they can be reused?
+
+/**
+ * Add UTM params to a URL
+ *
+ * @param string $url the URL
+ * @return string the URL with UTM params
+ */
+function add_utm_params( $url ) {
```

Comment:
@circlecube We should move all functions to static methods on the widget implementation class.

Reply:
Good call, got them moved to the main class in each case.